solitude/utils.py

```diff
@@ -2,6 +2,22 @@
 from django.core.exceptions import ImproperlyConfigured
 
 
+# Note deliberately ignoring o, O, 0, i and 1 as too similar.
+choices = 'abcdefghjklmnpqrstuvwxyzABCDEFGHJKLMNPQRSTUVWXYZ23456789'
+
+
+def shorter(integer):
+    """
+    Convert an integer into something shorter.
+    """
+    output = ''
+    while integer > 0:
+        remainder = integer % len(choices)
+        output = choices[remainder] + output
```

Comment:
if you did something more like:
````
output = []
output.append(choices[remainder])
return ''.join(output)
````
then I think it would be noticeably faster for bigger numbers. Same amount of code.